build-profiling-ffi.sh

```diff
@@ -100,15 +100,17 @@ sed < cmake/DatadogConfig.cmake.in \
 
 cp -v LICENSE LICENSE-3rdparty.yml NOTICE "$destdir/"
 
-export RUSTFLAGS="${RUSTFLAGS:- -C relocation-model=pic}"
 
 datadog_profiling_ffi="datadog-profiling-ffi"
 FEATURES="--features cbindgen,datadog-profiling-ffi/ddtelemetry-ffi"
 if [[ "$symbolizer" -eq 1 ]]; then
     FEATURES="--features cbindgen,datadog-profiling-ffi/ddtelemetry-ffi,symbolizer"
 fi
 
-DESTDIR="$destdir" cargo build --package="${datadog_profiling_ffi}" ${FEATURES} --release --target "${target}"
+# build inside the crate to use the config.toml file
+pushd profiling-ffi
+DESTDIR="$destdir" cargo build ${FEATURES} --release --target "${target}"
+popd
```

Comment:
Would it be cleaner to 
```suggestion
( cd profiling-ffi && DESTDIR="$destdir" cargo build ${FEATURES} --release --target "${target}")
```